packages/expo-sqlite/ios/EXSQlite.podspec

```diff
@@ -0,0 +1,23 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name           = 'EXSQlite'
```

Comment:
Change all `EXSQlite` occurrences to `EXSQLite` (+ rename `expo-sqlite/ios/EXSQlite` to `EXSQLite`) 🙂